halotools/mock_observables/large_scale_density/large_scale_density_spherical_annulus.py

```diff
@@ -105,9 +105,9 @@ def large_scale_density_spherical_annulus(sample, tracers, inner_radius, outer_r
             period, sample_volume, num_threads, approx_cell1_size)
         )
 
-    _ = npairs_per_object_3d(sample, tracers, rbins, period=period,
+    result = npairs_per_object_3d(sample, tracers, rbins, period=period,
```

Comment:
I think this, and one or two other similar changes, are the only examples of total pedantry. Seeing `_` used confused me just long enough. Sorry...

Reply:
Ha! Fine.